test/check-application

```diff
@@ -72,6 +72,23 @@ class TestApplication(testlib.MachineCase):
         # check exited alpine not in running list
         b.wait_not_in_text("#containers-containers", "alpine:latest")
 
+        # delete running container busybox using force delete
+        b.click('#containers-containers tbody tr:contains("busybox:latest") + tr button.btn-delete')
+        self.confirm_modal("btn-danger")
+        self.confirm_modal("btn-danger")
```

Comment:
Having to confirm two consecutive dialogs about the same thing feels strange to me. Of course this behaviour can be fixed in a follow-up PR, i. e. only have a "delete" or "force delete" dialog, not both.

However, in this case the test should make some assertions about the dialog, like verifying the primary button label ("Delete" or "Force Delete"). You could pass the expected button label as an argument to `confirm_modal()`.